### PR TITLE
Adding browser update script to site

### PR DESCRIPTION
**Pull request by FlynnFarrow**
Due to incompatibility issues with internet explorer 11 (basically nothing comes up except for a browser update script I've added) and some people using outdated opera/chrome/edge/firefox/safari versions I have added a browser update script which tells people to update their browser to the latest version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,6 +30,17 @@
     <!-- font choice from: https://github.com/JoseDelPino/dodo_passboard -->
     <!--<script type="text/javascript" src="lib/touch-emulator.js"></script>
     <script> TouchEmulator(); </script>-->
+    <!--Browser update script, minimum edge/ie browser is edge 79 (chromium) minimum firefox browser is 45, opera 45, safari 2 versions behind latest, chrome 60-->
```
Comment (FlynnFarrow) [inline index.html]: Browser update script below.

File: index.html [hunk 2]
```diff
@@ -30,6 +30,17 @@
...
+    <script> 
+var $buoop = {required:{e:79,f:45,o:45,s:-2,c:60},insecure:true,unsupported:true,api:2020.03 }; 
+function $buo_f(){ 
+ var e = document.createElement("script"); 
+ e.src = "//browser-update.org/update.min.js"; 
+ document.body.appendChild(e);
+};
+try {document.addEventListener("DOMContentLoaded", $buo_f,false)}
+catch(e){window.attachEvent("onload", $buo_f)}
+</script>
```
Comment (FlynnFarrow) [inline index.html]: Update script ended

**Comment by FlynnFarrow** [comment]
ie11outdatedbrowserscript

**Comment by FlynnFarrow** [comment]
Don't add the latest commit, I was doing Google Site verification.

**Comment by FlynnFarrow** [comment]
Again don't add the latest commit, changed meta tags to point to my page instead of your page.

**Comment by eugeneration** [comment]
This looks pretty useful - let me know when it might be ready!

**Comment by FlynnFarrow** [comment]
It is basically ready but I added some custom changes that's not needed e.g. 1st commit after I started the request was some Google Search Console stuff and 2nd one was changing meta tags which is not needed here. Best using commit #ced117e. I could also make a mobile app using MIT App Inventor/Kodular and maybe a desktop app with Visual Studio.

**Comment by FlynnFarrow** [comment]
If your looking for an android app I got an example.

Test it here: https://appetize.io/app/na02ktp3947011qddhuq5wytwm?device=pixel4&scale=50&orientation=portrait&osVersion=10.0&deviceColor=black

APK cannot be attached due to GitHub limitations, see of what you think about the app.

**Comment by ssteigen** [comment]
If there's a commit in this PR that is extra, @FlynnFarrow  you need to remove it.

**Comment by FlynnFarrow** [comment]
I managed to make a newer version and I got the app in a ZIP to put on your android phone.

[HappyIslandDesigner 1.01b.apk.zip](https://github.com/eugeneration/HappyIslandDesigner/files/4404256/HappyIslandDesigner.1.01b.apk.zip)
[Online app test](https://appetize.io/app/jy0y0ce9qzcdn7paw514aq0ny4
2020-03-30 18-04-58 Screenshot
)

**Comment by FlynnFarrow** [comment]
> If there's a commit in this PR that is extra, @FlynnFarrow you need to remove it.

How can I remove it from the pull request?

**Comment by FlynnFarrow** [comment]
.

**Comment by eugeneration** [comment]
I considered making a simple webview app for mobile but I think the current site does not meet user expectations of a native app. At minimum I feel it should have a menu that allows for selecting between multiple locally saved maps.

File: index.html [hunk 3]
```diff
@@ -30,6 +30,17 @@
...
```
Comment (ssteigen) [inline index.html:43]: This PR cannot be merged until these lines are the only changes present in the PR.

**Comment by FlynnFarrow** [comment]
@ssteigen as you kinda did notify @eugeneration think it's safe to close.

**Pull request closed without merging**